datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -184,19 +183,13 @@ def config(ctx, check, sync, verbose):
 
 
 def validate_default_template(spec_file):
-    init_config_default = False
-    instances_default = False
     if 'template: init_config' not in spec_file or 'template: instances' not in spec_file:
         # This config spec does not have init_config or instances
         return True
-
     for line in spec_file.split('\n'):
-        if any("init_config/{}".format(template) in line for template in TEMPLATES):
-            init_config_default = True
-        if any("instances/{}".format(template) in line for template in TEMPLATES):
-            instances_default = True
-
-        if instances_default and init_config_default:
+        if any("init_config/{}".format(template) in line for template in TEMPLATES) and any(
```

Comment:
suggestion: it could be more readable / easier to understand if this line is extracted to a method or a locally defined helper method.

Reply:
The function name would be descriptive enough to explain what we're trying to do. We probably wouldn't need to jump to the function definition.

Reply:
Exactly haha maybe one variable for the entire condition? or a comment in the worst case ?